### ISSUE TITLE
Natural dep residual - crash

### ISSUE THREAD

**Issue by thomscoder** [nearform/lyra#62]
**Describe the bug**
I was creating examples for #24 and noticed a crash on `lyra beta 16`

**To Reproduce**
```js
import { create, insert } from '@nearform/lyra';

const db = create({
  schema: {
    author: 'string',
    quote: 'string'
  }
});

insert(db, {
  quote: 'If you really look closely, most overnight successes took a long time.',
  author: 'Steve Jobs'
});

insert(db, {
  quote: 'If you are not willing to risk the usual, you will have to settle for the ordinary.',
  author: 'Jim Rohn'
});


const searchResult = search(db, {
  term: 'if',
  properties: '*'
});

console.log(searchResult)
```

**Screenshots**
<img width="1437" alt="Schermata 2022-07-30 alle 11 35 44" src="https://user-images.githubusercontent.com/78874117/181904682-79ac5239-257d-4403-89bc-4f4ee8ff5131.png">

**Desktop (please complete the following information):**
 - OS: Monterey 12.4
 - Node: 14.16.0 <= x <= 18.7.0

**Comment by mateonunez** [comment]
Nice catch @thomscoder.

I have the same issue with a fresh installation.

**Comment by micheleriva** [comment]
I'm having the same problem, which is kinda strange as you won't find any `Natural` reference in the codebase. This dependency has been removed, the package has been rebuild, but it still appears. Any idea?

**Comment by mateonunez** [comment]
I've fixed this bug by removing the `dist` folder and rebuilding only the **lyra package**. @micheleriva

**Comment by micheleriva** [comment]
Fixing it right now

**Comment by micheleriva** [comment]
Fixed in `0.0.1-beta-17`, thanks for the heads up!
